### DISCUSSION TITLE
Coders fusion

### DISCUSSION THREAD

**Discussion by fxstbg** [ccbogel/QualCoder#802] [Q&A] [answered]
It's my first work using QualCoder, and I've begun using it as "default" coder. After a while, I've entered my name so as my student can also code on the database. So, there are two coders ("default" and "FX") for the same user. 

Is there a way to merge the to coders into one ?

Thx again for this great app !!!

FX

**Comment by ccbogel** [comment, marked as answer]
OK, keep a backup copy first, then, you need to go to Reports > Database queries, to update the coder name everywhere using these SQL statements.
Run these statements one at a time (using the cogs button) to remove the FX name from these tables:
update  code_text set owner="default"
update project set codername="default"
update  code_image set owner="default"
update  code_av set owner="default"
update  code_name set owner="default"
update  code_cat set owner="default"
update source set owner="default"
update cases set owner="default"
update case_text set owner="default"
update annotation set owner="default"
update attribute set owner="default"
update  attribute_type set owner="default"
update journal set owner="default"

A possible warning error (in code_text) might occur if 'default' and 'FX' have coded the exact same text segment with the same code. As duplicates are not allowed to be coded by one coder. Probably won't happen but if it does let me know, as would need to delete one of the potentially duplicate code_text entries.
